### ISSUE TITLE
Expected indentation of 2 spaces

### ISSUE THREAD

**Issue by ravinggenius** [stylelint/stylelint#2646]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

I believe this is a bug. When linting [styled-components](https://www.styled-components.com/) styles inside a helper function, indentation is reported incorrectly.

> Which rule, if any, is this issue related to?

`indentation`

> What CSS is needed to reproduce this issue?

```javascript
import { css } from 'styled-components';

const helper = (condition) => {
  if (condition) {
    return css`
      color: red;

      &:hover {
        color: blue;
      }
    `;
  }
  return null;
};
```

> What stylelint configuration is needed to reproduce this issue?

```json
{
  "processors": ["stylelint-processor-styled-components"],
  "extends": "stylelint-config-standard",
  "syntax": "scss"
}
```

> Which version of stylelint are you using?

`7.11.1`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

`yarn run lint` which calls `stylelint 'src/**/*.js?(x)'`

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

Yes. Both styled-components and SCSS look for `&` as a placeholder when expanding nested rules.

> What did you expect to happen?

No errors to be reported. An error only gets reported when both the `color: red;` style and the `&:hover` block is present. Removing either does not trigger an `indentation` error to be reported.

> What actually happened (e.g. what warnings or errors you are getting)?

The following errors were flagged:

```shell
Button.jsx
 36:5  ✖  Expected indentation of 2 spaces   indentation
 38:5  ✖  Expected indentation of 2 spaces   indentation
 39:7  ✖  Expected indentation of 4 spaces   indentation
 40:5  ✖  Expected indentation of 2 spaces   indentation
 41:3  ✖  Expected indentation of 0 spaces   indentation
```

**Comment by jeddy3** [comment]
@ravinggenius Thank you for using template. However, this looks to be an issue with a 3rd party processor. I recommend opening an issue in [`stylelint-processor-styled-components` issue tracker](https://github.com/styled-components/stylelint-processor-styled-components/issues).

**Comment by ravinggenius** [comment]
Of course it is. Thanks anyway. I'll check over there.
